apps/examples/testcase/ta_tc/media/itc/itc_media_fileinputdatasource.cpp

```diff
@@ -64,9 +73,9 @@ static void itc_media_FileInputDataSource_getChannels_setChannels_p(void)
 {
 	media::stream::FileInputDataSource source(file_path);
 	TC_ASSERT_EQ("open", source.open(), 1);
-	TC_ASSERT_EQ("getChannels", source.getChannels(), 2);
+	TC_ASSERT_EQ_CLEANUP("getChannels", source.getChannels(), 2, source.close());
```

Comment:
Please use existing variables for value comparison:
static unsigned int channels = 2;
static unsigned int sampleRate = 16000;